### PR TITLE
compile pass

### LINKED ISSUE #1544: gie sdk refactor
as titled

### PR DESCRIPTION
**Pull request by tianliplus**
## Related issue number

Fixes #1544

### CODE REVIEW

File: interactive_engine/compiler/src/main/java/com/alibaba/maxgraph/compiler/tree/BranchTreeNode.java [hunk 1]
```diff
@@ -1,16 +1,14 @@
...
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
  * limitations under the License.
  */
 package com.alibaba.maxgraph.compiler.tree;
```
Comment (longbinlai) [inline interactive_engine/compiler/src/main/java/com/alibaba/maxgraph/compiler/tree/BranchTreeNode.java:14]: I think `maxgraph` better be replace with graphscope.gie, namely com.alibaba.graphscope.gie...

**Review by longbinlai — COMMENTED** [review]
Is it a better idea to include an sdk-common-v2 that include all the new codes without touching the legacy ones? While we complete the replacement of maxgraph with gaia, we then remove old sdk-common @yecol @tianliplus

File: interactive_engine/compiler/src/main/java/com/alibaba/maxgraph/compiler/tree/BranchTreeNode.java (hunk 1 above)
Comment (yecol) [reply interactive_engine/compiler/src/main/java/com/alibaba/maxgraph/compiler/tree/BranchTreeNode.java:14]: Yes, I cannot agree more. 

Actually, I also found these in shirly121's PR and recorded in #1499 (the last line).

**Comment by yecol** [comment]
sdk-common-v2 is a good idea! 
I agree!

**Comment by tianliplus** [comment]
There are a lot of "com.aliababa.maxgraph" in our project. I think we should use a dedicated issue to track the refactoring work. see #1564

**Review by longbinlai — APPROVED** [review]
Another issue is created for renaming the package of sdk-common.

**Pull request merged**